cmd/init.go

```diff
@@ -38,6 +39,45 @@ var initCommand = &cobra.Command{
 			os.Exit(1)
 		}
 
+        // Create Git hook for Skynet Upload [dg-pages]
+        b := []byte(`#!/bin/bash
+            # Upload current directory to skynet if branch is dg-pages
+
+            echo "Running update hook"
+
+            remote=$1
+            remote_url="$2"
+
+
+            echo "Remote name is $remote"
+            echo "Remote URL is $remote_url"
+
+            while read local_ref remote_ref
+            do
+                echo "Local branch $local_ref"
```

Comment:
Given this hook executes on every push, I would suggest removing these trace/debug lines from 46, 52-43, and 57. Optionally you can wrap them in a `"$DGIT_LOG_LEVEL" == "debug"` conditional

Reply:
I totally forgot about those. Removed all echoes